examples/ITR_UI.py

```diff
@@ -821,7 +853,10 @@ def recalculate_individual_itr(warehouse_pickle_json, eibm, proj_meth, winz, bm_
         Warehouse.company_data.projection_controls.TREND_CALC_METHOD = ITR_median if proj_meth == 'median' else ITR_mean
         Warehouse.company_data.projection_controls.LOWER_PERCENTILE = winz[0] / 100
         Warehouse.company_data.projection_controls.UPPER_PERCENTILE = winz[1] / 100
-
+    
```

Comment:
Suggest moving this elsewhere (calculation of temperature score) per the above.

Reply:
I moved this part to the function calc_temperature_score.